### PR TITLE
[DOC-306] [DOC-320] [DOC-357] [yba] 2024.1 Release management enhancement, 2024.1 version naming changes.

### PR DESCRIPTION
**Pull request by aishwarya24**
Release management updates
DOC-306: Update YBA docs for Releases feature
DOC-320: Add 2024.1 version naming changes to docs
DOC-357: New Stable Release numbering (2024.1)

@netlify /preview/yugabyte-platform/manage-deployments/upgrade-software/

### CODE REVIEW

File: docs/content/preview/yugabyte-platform/configure-yugabyte-platform/kubernetes.md [hunk 1]
```diff
@@ -379,7 +379,7 @@ If you plan to create multi-region YugabyteDB universes, you can set up [Multi-C
 
 The Kubernetes MCS API is currently in alpha, though there are various implementations of MCS which are [considered to be stable](https://github.com/kubernetes-sigs/mcs-api/issues/17#issuecomment-1309073682). To know more, see [API versioning](https://kubernetes.io/docs/reference/using-api/#api-versioning) in the Kubernetes documentation.
 
-MCS support in YBA is currently in [Early Access](../../../releases/versioning/#feature-availability). Keep in mind following caveats:
+MCS support in YBA is currently in [Early Access](../../../../releases/versioning/#feature-maturity). Keep in mind following caveats:
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/configure-yugabyte-platform/kubernetes.md:382]:
```suggestion
MCS support in YBA is currently in [Early Access](/preview/releases/versioning/#feature-maturity). Keep in mind following caveats:
```

File: docs/content/preview/yugabyte-platform/upgrade/upgrade-yp-installer.md [hunk 2]
```diff
@@ -32,6 +32,8 @@ type: docs
 
 If your YugabyteDB Anywhere installation was installed using [YBA Installer](../../install-yugabyte-platform/install-software/installer/), use YBA Installer to upgrade to a newer version. Note that you can only upgrade to a newer version; downgrades are not supported.
 
+For more information, refer to [Compatibility with YugabyteDB](../../../releases/yba-releases/#compatibility-with-yugabytedb).
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-platform/upgrade/upgrade-yp-installer.md:35]:
```suggestion
For more information, refer to [Compatibility with YugabyteDB](/preview/releases/yba-releases/#compatibility-with-yugabytedb).
```

File: docs/content/preview/yugabyte-voyager/performance.md [hunk 3]
```diff
@@ -56,7 +56,7 @@ Use one or more of the following techniques to improve import data performance:
 
 - **Add disks** to reduce disk write contention. YugabyteDB servers can be configured with one or multiple disk volumes to store tablet data. If all tablets are writing to a single disk, write contention can slow down the ingestion speed. Configuring the [YB-TServers](../../reference/configuration/yb-tserver/) with multiple disks can reduce disk write contention, thereby increasing throughput. Disks with higher IOPS and better throughput also improve write performance.
 
-- **Enable packed rows** ([Early Access](../../releases/versioning/#feature-availability)) to increase the throughput by more than two times. Enable packed rows on the YugabyteDB cluster by setting the YB-TServer flag [ysql_enable_packed_row](../../reference/configuration/yb-tserver/#ysql-enable-packed-row) to true.
+- **Enable packed rows** ([Early Access](../../releases/versioning/#feature-maturity)) to increase the throughput by more than two times. Enable packed rows on the YugabyteDB cluster by setting the YB-TServer flag [ysql_enable_packed_row](../../reference/configuration/yb-tserver/#ysql-enable-packed-row) to true.
```
Comment (ddhodge) [inline docs/content/preview/yugabyte-voyager/performance.md:59]:
```suggestion
- **Enable packed rows** ([Early Access](/preview/releases/versioning/#feature-maturity)) to increase the throughput by more than two times. Enable packed rows on the YugabyteDB cluster by setting the YB-TServer flag [ysql_enable_packed_row](../../reference/configuration/yb-tserver/#ysql-enable-packed-row) to true.
```

File: docs/content/preview/faq/yugabytedb-managed-faq.md [hunk 4]
```diff
@@ -146,12 +146,12 @@ For more information, refer to [Maintenance windows](../../yugabyte-cloud/cloud-
 
 ### What version of YugabyteDB does my cluster run on?
 
-Dedicated clusters are provisioned with a **stable** release, from a YugabyteDB [stable release](../../releases/versioning/#stable-releases) series. When creating a dedicated cluster, you can choose one of the following tracks:
+Dedicated clusters are provisioned with a **stable** release, from a YugabyteDB [stable release](../../releases/versioning/#release-versioning-convention-for-stable-releases) series. When creating a dedicated cluster, you can choose one of the following tracks:
```
Comment (ddhodge) [inline docs/content/preview/faq/yugabytedb-managed-faq.md:149]:
```suggestion
Dedicated clusters are provisioned with a **stable** release, from a YugabyteDB [stable release](/preview/releases/versioning/#release-versioning-convention-for-stable-releases) series. When creating a dedicated cluster, you can choose one of the following tracks:
```

File: docs/content/preview/faq/yugabytedb-managed-faq.md [hunk 5]
```diff
@@ -146,12 +146,12 @@ For more information, refer to [Maintenance windows](../../yugabyte-cloud/cloud-
...
 
 - Innovation, which has more frequent updates for faster access to new features.
 - Production, which has a slower update cadence and features only select stable release builds.
 
-In addition to the Innovation track, Sandbox clusters can be provisioned with a **preview** release, from the YugabyteDB [preview release](../../releases/versioning/#preview-releases) series.
+In addition to the Innovation track, Sandbox clusters can be provisioned with a **preview** release, from the YugabyteDB [preview release](../../releases/versioning/#release-versioning-convention-for-preview-releases) series.
```
Comment (ddhodge) [inline docs/content/preview/faq/yugabytedb-managed-faq.md:154]:
```suggestion
In addition to the Innovation track, Sandbox clusters can be provisioned with a **preview** release, from the YugabyteDB [preview release](/preview/releases/versioning/#release-versioning-convention-for-preview-releases) series.
```

File: docs/content/preview/releases/versioning.md [hunk 6]
```diff
@@ -44,9 +45,22 @@ Patch and revision releases in a stable release series (`MAJOR.EVEN`) include bu
 
 {{< /note >}}
 
-### Preview releases
+## Release versioning convention for Preview releases
+
+Preview releases, which include features under active development, are recommended for development and testing only.
+
+YugabyteDB follows the below-described convention for preview release version numbering. Preview release versions follow the versioning format of `MAJOR.MINOR.PATCH.REVISION`, where non-negative integers are used for:
```
[the remark this answers is not in the document]
Comment (aishwarya24) [reply docs/content/preview/releases/versioning.md]: hotfix

**Pull request merged**